src/test/ifaddrs/test_ifaddrs.rs

```diff
@@ -15,7 +15,7 @@ fn main() {
 
     let mut ip_vec = vec![];
     for ifaddr in addrs {
-        if let Some(nix::sys::socket::SockAddr::Inet(address)) = ifaddr.address {
+        if let Some(address) = ifaddr.address {
             let address_str = address.to_string();
             let ip = address_str.split(":").collect::<Vec<&str>>()[0];
             println!(
```

Comment:
I think this code will have weird behaviour if it's an IPv6 address (for example where colons in an IPv6 address like `fe80::1ff:fe23::` aren't used to split the port). Maybe this should have an extra `if let Some(address) = address.as_sockaddr_in() {...}` check?

Reply:
Done. Rearranged a bit to avoid excessive nesting.